workflow.txt

```diff
@@ -1,10 +1,10 @@
+fork on github
 git clone
-git checkout -b my-solution
+git add .
 git commit -m ""
```

Comment:
Once our files are committed to our new branch, we can switch back to master
`git checkout master`
While working on our branch other developers might have pushed changes to the remote, so we pull those changes down
`git pull upstream master`
now we can merge our branch into master
`git merge my-solution`
and finally push our changes to the remote branch
`git push origin my-solution`